cluster/src/main/java/org/apache/iotdb/cluster/partition/slot/SlotPartitionTable.java

```diff
@@ -103,7 +104,7 @@ public static void setSlotStrategy(SlotStrategy slotStrategy) {
   private void init(Collection<Node> nodes) {
     logger.info("Initializing a new partition table");
     nodeRing.addAll(nodes);
-    nodeRing.sort(Comparator.comparingInt(Node::getNodeIdentifier));
```

Comment:
so, if we sort node by ip:port, it's difficult to support ip or port change.

Reply:
yes, If the IP address and port are bound, the modification is difficult.

Reply:
OK, I will remove this.